addon/convert-schema.js

```diff
@@ -0,0 +1,200 @@
+import _ from 'lodash'
+
+function pathFinder (valueObj, prevPath) {
+  return function (path) {
+    if (!_.isArray(path)) {
+      path = path.split('.').reverse()
+    }
+    let nextInPath = _.last(path)
+
+    if (nextInPath === '') {
+      path.pop()
+      if (_.last(path) === '') {
+        path.pop()
+        path.push(path.pop().replace('/', ''))
+        return prevPath(path)
+      } else {
+        path.push(path.pop().replace('/', ''))
+      }
+    }
+    return _.get(valueObj, path.reverse().join('.'))
+  }
+}
+
+// (value, condition)->boolean
+function meetsCondition (value, condition) {
+  let isConditionMet = false
+
+  if (condition.equals) {
+    isConditionMet = isConditionMet || _.isEqual(condition.equals, value)
+  }
+
+  if (condition.greaterThan) {
+    isConditionMet = isConditionMet || value > condition.greaterThan
+  }
+
+  if (condition.lessThan) {
+    isConditionMet = isConditionMet || value < condition.lessThan
+  }
+
+  return isConditionMet
+}
```

Comment:
We will end up having more than these 3 conditions and I don't want this single conditional to get too large. I'll see if I can streamline this and get it closer to a more permanent solution.